diff --git a/dist/pages/runtime/plugins/check-if-page-unused.js b/dist/pages/runtime/plugins/check-if-page-unused.js
index d5d01db3367222689db7aa1118c0d38450c85d57..7e6f6280b683f4aa46ad50b854a59debfc90dd35 100644
--- a/dist/pages/runtime/plugins/check-if-page-unused.js
+++ b/dist/pages/runtime/plugins/check-if-page-unused.js
@@ -56,4 +56,5 @@ const plugin = defineNuxtPlugin({
 	env: { islands: false }
 });
 //#endregion
-export { NESTED_PAGE_CONFIRMATION_DELAY, plugin as default, findUnrenderedNestedPage };
+export { NESTED_PAGE_CONFIRMATION_DELAY, findUnrenderedNestedPage };
+export default plugin;
